features/networking/ovn-egress-ip.feature

```diff
@@ -0,0 +1,68 @@
+Feature: OVN Egress IP related features
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-33618
+  @admin
+  @destructive
+  Scenario: EgressIP works for all pods in the matched namespace when only configure namespaceSelector
+    Given I save ipecho url to the clipboard
```

Comment:
so this mean this case only can be tested on vsphere cluster for now. 
if so, please also update case filter.

Reply:
Yes, this auto case can only be run in vSphere.  Will update the case filter in polarion.  Thanks for reminder.